security/pinact/Portfile

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/suzuki-shunsuke/pinact 2.1.0 v
+categories          security
+maintainers         {macports.halostatue.ca:austin @halostatue} \
+                    openmaintainer
+license             MIT
+
+description         A CLI to edit GitHub Workflow and Composite action files and pin \
+                    versions of Actions and Reusable Workflows. pinact can also update \
+                    their versions and verify version annotations.
+
+long_description    {*}${description}
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  823b219697a24a9d587f4754bfb4bc80642b402a \
+                    sha256  1fcd2a45767aedb5cff4de72cbb8acacee0d841641d0dd8c2b787c3fb47b8d30 \
+                    size    30146
+
+set time [clock format [clock seconds] -format "%Y-%m-%d"]
```

Comment:
Would probably be best to set this to some fixed timestamp to better align towards reproducible builds.

Reply:
Sure. I can't find where the date is output, so I can either omit it or put it as a fixed date when updates are done (they already have an update from creating this last night).

Reply:
I personally usually opt to omit, but unfortunately one can't be sure if that could cause something to crash down the line.

Reply:
It won't cause a crash because it's not a pointer. It's also not used by pinact (took too long to figure that out), so I’m omitting it.